variables/typography.scss

```diff
@@ -23,83 +23,79 @@
 
   @return map-get($typography-map, size);
 }
-// Keep until systems are updated with css variable. 
+
 $font-family: 'Roboto', Arial;
-// As CSS variable
-:root {
-  --hdd-font-family: $font-family;
-}
```

Comment:
Hvorfor fjernet?

Reply:
Fikk en litt merkelig feil når variabelen ble brukt mot p tagger. Da gikk hele siden over til Times, så bare fjernet den foreløpig. Men legger opp variabelen nå, men definerer den bare direkte med fonten isteden for å peken mot scss variabelen, da fungerer det fint.